modules/logging/pom.xml

```diff
@@ -37,10 +37,12 @@
       <groupId>javax.inject</groupId>
       <artifactId>javax.inject</artifactId>
     </dependency>
+    <!-- 
```

Comment:
Don't keep dependencies commented out on the code if they have no "didactic" purpose or if they are not a "placeholder" for extending the code